saleor/core/permissions.py

```diff
@@ -17,6 +17,8 @@
     'user.edit_user',
     'voucher.view_voucher',
     'voucher.edit_voucher',
+    'group.view_group',
+    'group.edit_group',
```

Comment:
You added view_group and edit_group to user model so it should be user.view_group and user.edit_group

Reply:
Ok, added also `.view_staff`.